CHANGELOG.md

```diff
@@ -66,6 +66,20 @@ Swift Next
   let _: Foo? = .bar.anotherFoo.getFoo().optionalFoo?.optionalFoo![]
   ```
 
+* [SE-0284][]:
```

Comment:
Nit: This needs to go above SE-0287. Looks like “newer entries on top” text that used to be there has disappeared...

Reply:
Looks like the text is still there above the table, I just didn't see it and forgot. Thanks!